lib/assessment.rb

```diff
@@ -5,7 +5,9 @@ class Assessment
   # enumerable supplied.  Assume each entry implements a length
   # method that returns a Fixnum.
   def longest(enum)
-    # Your code goes here
+    a_word = %w(big ass word here)
+    a_word.max
+    a_word.max { |a, b| a.length <=> b.length }
```

Comment:
No, not quite. You've got the right block; you just need to pass it into the right method on `enum`. Can you figure out which it is?